### ISSUE TITLE
Decide where to enforce missing content from CPS pages

### ISSUE THREAD

**Issue by dr3** [bbc/simorgh#4406]
**Is your feature request related to a problem? Please describe.**
If an image is missing alt text, should the preprosessors or the containers stop the render?

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**
